vscode/src/modelProvider/__tests__/scripts/genCerts.sh

```
@@ -0,0 +1,31 @@
+set -euo pipefail
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)